src/components/Pages/Guide/GuideSectionCollection.js

```diff
@@ -64,14 +58,15 @@ function GuideSection({
             departments,
             slug,
           })}
+          target="_blank"
```

Comment:
this is the only thing that actually changed, everything else is just jsprettier formatting